docs/source/guides/command-groups.rst

```diff
@@ -148,5 +145,5 @@ Extra Information
 =================
 
 By default, subcommands and subcommand groups **will not** inherit the checks from the parent group or subgroup that the command
-belongs to. This behaviour can be changed by providing the kwarg ``inherit_checks=True`` in the :obj:`lightbulb.decorators.command`
+belongs to. This behaviour can be changed by providing the kwarg ``inherit_checks=True`` in the :obj:`lightbulb.command`
```

Comment:
```suggestion
belongs to. This behaviour can be changed by providing the kwarg ``inherit_checks=True`` in the :obj:`lightbulb.decorators.command`
```
This has to stay like this otherwise I don't think the docs will link correctly.

Reply:
yep!